src/components/TableView/common/Sort.ts

```diff
@@ -0,0 +1,13 @@
+import { DateTime } from 'luxon';
+
+export enum SortDirection {
+  ASC = 'ASC',
+  DESC = 'DESC',
+}
+
+export interface Sort {
+  direction: SortDirection | undefined;
+  column: number;
```

Comment:
Why is it number?

Reply:
cuz we don't have column names, i store in inner sort value column index